test/PushComm/unit_tests/SendingNotifications/SendNotifs.t.sol

```diff
@@ -19,10 +15,6 @@ contract SendNotifs_Test is BaseTest {
         assertEq(res, false);
     }
 
-    function test_WhenRecipientIsSendingNOTIFOnlyToThemselves() external {
-        //User can no longer send notifs to themselves
-    }
-
     function test_WhenChannelIs0x00ButCallerIsAnyAddressOtherThanAdminOrGovernance() external {
```

Comment:
The test names are a bit to lengthy and unreadable - `test_WhenChannelIs0x00ButCallerIsAnyAddressOtherThanAdminOrGovernance()` 

Making changes to make it readable. @Zartaj0